fact-bounty-flask/tests/test_register.py

```diff
@@ -29,7 +29,7 @@ def test_can_register(self):
         response = self.app.post('/api/users/register', data=json.dumps(dict(name=USER_DATA['name'],email=USER_DATA['email'], password=USER_DATA['password'],password2=USER_DATA['password2'])),content_type='application/json', follow_redirects=True)
         res = response.data.decode('ASCII')
         res = json.loads(res)
-        self.assertEqual(response.status_code, 201)
+        self.assertEqual(response.status_code, 202)
         self.assertEqual(res['message'], 'You registered successfully. Please log in.')
```

Comment:
The status_code is correct as test user is not deleted from the database that's why this particular test is failing I am working on the test_user deletion part and will make the PR soon

Reply:
![Screenshot from 2019-03-19 18-03-24](https://user-images.githubusercontent.com/34132255/54606308-7bc86000-4a71-11e9-9082-bca6708090c4.png)
You can see in the above image :slightly_smiling_face: